config.json

```diff
@@ -175,7 +175,7 @@
           "path": "Tools/msbuild.cmd"
         },
         "unix":{
-          "defaultParameters": "/nologo /verbosity:minimal /clp:Summary /maxcpucount /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log",
+          "defaultParameters": "/nologo /verbosity:minimal /clp:Summary /maxcpucount /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log /p:SkipTests=true",
```

Comment:
Moved to the dir.props file, opened https://github.com/dotnet/buildtools/issues/1430 and added a comment.